### PR TITLE
dark mode + light/dark toggle

### PR DESCRIPTION
**Pull request by blakeheyman**
Create a toggle to switch between light and dark mode.

### CODE REVIEW

File: src/SettingsMenu.tsx [hunk 1]
```diff
@@ -4,16 +4,35 @@ import React from "react";
...
+  const [openInDiscord, setOpenInDiscord] = useLocalStorage("openInDiscord", true);
   return (
     <div>
+      <Row align="middle" gutter={[10, 0]}>
+        <Col>Dark Mode</Col>
+        <Col>
+          <Switch checked={currentTheme == 'dark' || cookies.currentTheme == null ? true : false} onChange={toggleTheme} />
+        </Col>
+      </Row>
       <Row align="middle" gutter={[10, 0]}>
```
Comment (blakeheyman) [inline src/SettingsMenu.tsx]: Can you add a `<SettingsDivider />` here

File: src/SettingsMenu.tsx [hunk 2]
```diff
@@ -4,16 +4,35 @@ import React from "react";
...
 
 const SettingsDivider = styled(Divider)({
   margin: "10px 0",
 });
 
 const SettingsMenu = (): JSX.Element => {
-  const [openInDiscord, setOpenInDiscord] = useLocalStorage("openInDiscord", true);
 
+  const { switcher, themes } = useThemeSwitcher();
+  const [cookies, setCookie] = useCookies(['currentTheme']);
```
Comment (blakeheyman) [inline src/SettingsMenu.tsx]: Q: Is there a reason to use cookies over LocalStorage here?

File: src/SettingsMenu.tsx [hunk 3]
```diff
@@ -4,16 +4,35 @@ import React from "react";
...
+  var currentTheme = cookies.currentTheme;
```
Comment (blakeheyman) [inline src/SettingsMenu.tsx]: Nit: Should be `const` or `let` if you need to reassign it.

File: src/App.tsx [hunk 4]
```diff
@@ -37,7 +53,12 @@ const App = (): JSX.Element => {
           Pokémon GO Raiders
         </Footer>
       </Layout>
+
+      </ThemeSwitcherProvider>
     </Router>
+
+  
+
```
Comment (blakeheyman) [inline src/App.tsx]:
```suggestion
```

File: src/App.tsx [hunk 5]
```diff
@@ -7,13 +7,29 @@ import SettingsMenu from "./SettingsMenu";
...
+import { getCookie } from 'cookied';
+
 const logo = require("./assets/pgricon64.png");
 
+const currThemes = {
+  dark: '/modomator/dark-theme.css',
+  light: '/modomator/light-theme.css'
+};
+
+const currentTheme = getCookie('currentTheme');
```
Comment (blakeheyman) [inline src/App.tsx]: Related to another comment: Is there a reason to cookies over LocalStorage here?

File: public/dark-theme.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  border-width: 1px !important;
+}
+
+form + div {
+  margin-top: 10px;
+}
+
+.ant-radio-button-wrapper-checked{
+border-color: white !important;
+border-width: 2px !important;
```
Comment (blakeheyman) [inline public/dark-theme.css]: Nit: The indentation looks a little weird here and elsewhere in the file.

File: src/App.tsx [hunk 7]
```diff
@@ -37,7 +53,12 @@ const App = (): JSX.Element => {
           Pokémon GO Raiders
         </Footer>
       </Layout>
+
+      </ThemeSwitcherProvider>
```
Comment (blakeheyman) [inline src/App.tsx:57]: Nit: I feel like the children of this element should be indented another level. Perhaps re-indent the whole file?

File: src/SettingsMenu.tsx (hunk 1 above)
Comment (Nathan-Meireson) [reply src/SettingsMenu.tsx]: added in next commit

File: src/SettingsMenu.tsx (hunk 3 above)
Comment (Nathan-Meireson) [reply src/SettingsMenu.tsx]: changed to let in next commit

File: public/dark-theme.css (hunk 6 above)
Comment (Nathan-Meireson) [reply public/dark-theme.css]: reformatted in next commit

File: src/App.tsx (hunk 4 above)
Comment (Nathan-Meireson) [reply src/App.tsx]: reformatted in next commit

File: src/App.tsx (hunk 7 above)
Comment (Nathan-Meireson) [reply src/App.tsx:57]: reformatted in next commit

File: src/App.tsx (hunk 5 above)
Comment (Nathan-Meireson) [reply src/App.tsx]: no, changed to LocalStorage in next commit

File: src/SettingsMenu.tsx (hunk 2 above)
Comment (Nathan-Meireson) [reply src/SettingsMenu.tsx]: no, changed to LocalStorage in next commit

**Pull request merged**